pkg/monitoring/monitoring.go

```diff
@@ -24,6 +24,7 @@ const (
 const (
 	cattleNamespaceName                    = "cattle-prometheus"
 	cattleCreatorIDAnnotationKey           = "field.cattle.io/creatorId"
+	EnablerAnno                            = "field.cattle.io/monitoringEnabler"
```

Comment:
Do we need another annotation to be introduced? If it's a system component, launched in the system project, set the creatorId to system user.